src/main/java/com/github/leroyguillaume/keycloak/bcrypt/BCryptPasswordHashProvider.java

```diff
@@ -17,7 +20,7 @@
     private final int defaultIterations;
     private final String providerId;
 
-    public BCryptPasswordHashProvider(String providerId, int defaultIterations) {
+    BCryptPasswordHashProvider(String providerId, int defaultIterations) {
```

Comment:
Why removing modifier?

Reply:
Just to get rid of an warning in the build tool :-)
Not important.